quantuminspire/cli/command_list.py

```diff
@@ -221,6 +223,16 @@ def sync_projects(
     typer.echo(f"Sync projects with {dest.value}")
 
 
+@files_app.command("upload")
+def upload_files(name: str = typer.Argument(..., help="The name of the file to upload")) -> None:
```

Comment:
Can the name be a full path to the file? What would the remote name be in that case?